src/index.ts

```diff
@@ -133,6 +133,10 @@ function createInstance(options: Options): Outdent {
         }
     }
 
+    (outdent as any).string = (str: string): string => {
```

Comment:
This will currently be breaking the build. I have absolutely no idea how to type this in TS. Halp.

Reply:
I'm a regular Flow user and, basically, what I want to express is something like this: https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVUCuA7AxgFwEs5sxcAnAUwEN9KAKa8gcwC4wAjOOGG7ASnYBnfOULZmYAD5gAKpRFgA3qjBhCUMIxb9lqsAEgq+TOVIBycwG59AX324SiuooC8W3a4B8YSzbUu+AB0ALaU+AAWcAAmYO70nj5++sampIE29qj4AJ4ADpRgAGKk8YlgImIS2fmF8m7FpABkemphkTHsJai2NkA.

Reply:
No worries, I'll make a commit to fix the types.

```
type Fn = () => string;
type Test = Fn & {
  method: Fn
};
```

That actually works in TypeScript; it was just balking at augmenting a function without first declaring the extended interface.  `(outdent as Outdent).string = ` does the trick.

One of these days I'll learn some Flow.

Reply:
Ok, that would explain my confusion then. So close!